pitkeel-py/test_daemon.py

```
@@ -0,0 +1,133 @@
+import sys
+from unittest.mock import MagicMock, call, patch
+
+import pytest
+
+
+# ---------------------------------------------------------------------------
+# _shutdown failure path tests
+# ---------------------------------------------------------------------------
+
+
+def test_shutdown_exits_nonzero_when_command_fails_twice(monkeypatch):
```

Ruff (0.15.5)

[warning] Unused function argument: `monkeypatch`

(ARG001)